docs/VnVPlan/VnVPlan.tex

```diff
@@ -347,6 +347,47 @@ \subsubsection{Area of Testing2}
 
 ...
 
+\subsubsection{Operational and Environmental Requirements}
+
+\begin{enumerate}
+\item{NFRT-OER1\\}
+
+Type: Functional, Dynamic, Manual
+
+Initial State: Application is not yet installed on the device.
+                
+Input: The user will install and run the application on various devices.
+                
+Output: The application will be successfully installed. The application is operational.
+                
+How test will be performed: The team will install the application on various mobile devices running Android on version 13 (IASR2) and iOS on version 16 (IASR1). The team will also test on various tablet devices running Android or iPadOS. The team will confirm that all functionalities work as expected on each device by conducting variety of actions and system events.
```

Comment:
For the last sentence, can you be more specific? We don't have to explain everything, but maybe specify major features for instance, "the tester will login and try to input receipt data and analyze spending analytics."

Reply:
resolved